### ISSUE TITLE
403 errors against /ims/api/incident_reports endpoint (for unattached reports)

### ISSUE THREAD

**Issue by wsanchez** [burningmantech/ranger-ims-server#86]
403 errors against /ims/api/incident_reports endpoint

Examples:
```
2016-07-22T13:24:50-0700 [twisted.python.log#info] "127.0.0.1" k8 - [22/Jul/2016:20:24:49 +0000] "GET /ims/api/incident_reports/?event=;incident= HTTP/1.0" 403 17 "https://khaki-shack.prettyok.net/ims/events/2015/incidents/498" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.106 Safari/537.36"
2016-07-26T10:24:44-0700 [twisted.python.log#info] "127.0.0.1" Splinter - [26/Jul/2016:17:24:44 +0000] "GET /ims/api/incident_reports/?event=;incident= HTTP/1.0" 403 17 "https://khaki-shack.prettyok.net/ims/events/Test/incidents/new" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.81 Safari/537.36 OPR/30.0.1835.59"
2016-07-26T10:25:03-0700 [twisted.python.log#info] "127.0.0.1" Splinter - [26/Jul/2016:17:25:03 +0000] "GET /ims/api/incident_reports/?event=;incident= HTTP/1.0" 403 17 "https://khaki-shack.prettyok.net/ims/events/Test/incidents/11" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_8_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.81 Safari/537.36 OPR/30.0.1835.59"
```

See with multiple user agents (Safari, Mobile Safari, Chrome).

This is asking for unattached reports: `?event=;incident=`

**Comment by wsanchez** [comment]
OK, logged in as `Easy E` I do get a 403 from `https://khaki-shack.prettyok.net/ims/api/incident_reports/?event=;incident=`

**Comment by wsanchez** [comment]
The problem here is that the `readIncidentReports` authz is not granted to anyone except admins.

**Comment by wsanchez** [comment]
Fixed: 31256336b451a046c5209c249395ced6b1210a66
